### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ryliecc**
add form page
update footer

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -49,6 +52,11 @@
           <li class="question-card__tag"><a href="Beispiellink3">#css</a></li>
         </ul>
       </section>
+      <!-- <script>
```
Comment (julialoeschel) [inline bookmarks.html:55]: always delete unused code for merge

File: form.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Rylie's Quiz App</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <script src="./index.js" defer></script>
+  </head>
+  <body>
+    <header><h1>Create new Questions</h1></header>
```
Comment (julialoeschel) [inline form.html:12]: if the `h1` is the only thing in a header, its possible to work without he `header`. h1 indicates the main heading for the page anyways.

File: form.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          rows="3"
+        ></textarea>
+        <label for="create__tag">Tag:</label>
+        <input type="text" id="create__tag" />
+        <button type="submit">Create question</button>
+      </form>
+    </main>
+    <footer class="navigation__footer">
+      <nav class="navigation__nav">
+        <a href="./index.html">
```
Comment (julialoeschel) [inline form.html:36]: if in a `a ` Tag there is no readable code, consider using aria-labels for better a11y

File: index.html [hunk 4]
```diff
@@ -7,6 +7,11 @@
     <title>Rylie's Quiz App</title>
     <link rel="stylesheet" href="./styles.css" />
     <script src="./index.js" defer></script>
+    <!-- <script>
+      let questionCardMarked = document.querySelector (class="question-card--marked");
+            let clonedQuestionCardMarked = questionCardMarked.cloneNode(true);
+            document.body.appendChild(clonedQuestionCardMarked);
+    </script> -->
```
Comment (julialoeschel) [inline index.html:14]: delete

File: index.js [hunk 5]
```diff
@@ -5,6 +5,7 @@ cards.forEach((card) => {
 
   bookmarkButton.addEventListener("click", () => {
     bookmarkButton.classList.toggle("bookmark-button--enabled");
+    /* cards.classList.toggle("question-card--marked"); */
```
Comment (julialoeschel) [inline index.js:8]: ;)

File: form.html (hunk 2 above)
Comment (ryliecc) [reply form.html:12]: I just checked if it makes sense to get rid of the 'header' but turns out I use them for styling reasons in css differently :)

**Pull request merged**